autopush/endpoint.py

```diff
@@ -342,9 +342,12 @@ def _store_auth(self, jwt, crypto_key, token, result):
     def _invalid_auth(self, fail):
         if isinstance(fail.value, VapidAuthException):
             raise fail.value
+        if isinstance(fail.value, JOSEError):
```

Comment:
This could be added to the isinstance check above, it can take a tuple of types to check.